### ISSUE TITLE
Paragraph 1.2.0-beta1 not compatible with Paragraphs jQuery UI Accordion

### ISSUE THREAD

**Issue by bdalomgir** [backdrop-contrib/paragraphs#127]
After updating the paragraphs 1.x-1.1.0 to 1.x-1.2.0-beta1 version it affects the paragraphs jQuery UI Accordion module. As a result, it breaks all the accordion designs.

**Comment by laryn** [comment]
Did you update all modules (eg. Entity plus) and clear caches? Can you provide more info on how it breaks? I am using that module without a problem in a few places I think.

**Comment by bdalomgir** [comment]
@laryn Thanks for your reply. Yes, we updated all modules and cleared caches.
Before upgrading the paragraph module it addressed all required accordion classes to the markup please see the screenshot below
![Screenshot from 2022-03-09 12-52-05 png (1906×577) (1)](https://user-images.githubusercontent.com/13948360/157390196-610d60a8-1d88-4fba-a0e8-c23fa011d1cd.png)

But after upgrading it does not include any accordion related classes to the markup. Screenshot below for reference - 
![Screenshot from 2022-03-09 12-53-15 png (1919×800)](https://user-images.githubusercontent.com/13948360/157390323-f25f7168-9b6a-4ae4-9554-9b16c98dc613.png)

**Comment by laryn** [comment]
Do you have a customized template file in your theme for accordion paragraphs? If so that might need slight tweaking.

**Comment by argiepiano** [comment]
I have done some testing with Paragraphs 1.x-1.2.0-beta1 and Paragraphs jQuery UI Accordion 1.x-1.0.1 and I can't reproduce this issue. The elements have the right classes, and the accordion seems to be working. Backdrop 1.21.3 and PHP 7.3.29.

Like @laryn, I also suspect that this issue may be the result of a customized paragraphs-item or node template file.  
<img width="889" alt="Screen Shot 2022-03-09 at 9 58 10 PM" src="https://user-images.githubusercontent.com/9938978/157592698-3ac6534e-2886-4079-8af2-04557f04c421.png">

**Comment by bdalomgir** [comment]
Thanks @laryn and @argiepiano for your reply. I have debugged again and found that normal accordion in paragraph working just fine. But we had a different case. We created an accordion using a title and description (which is another paragraph referenced). In this case its not working. Please check the screenshot below what we did - 
![Accordion-with-reference-paragraph](https://user-images.githubusercontent.com/13948360/157606672-1a9bc0e3-6f3b-490a-ae32-0ebe03bbda3e.png)
![paragraph-config-page](https://user-images.githubusercontent.com/13948360/157606679-7009d796-79d2-4093-8680-ced1dd35ff74.png)

**Comment by argiepiano** [comment]
When you say "not working" what exactly do you mean? Please provide more information about the issue.

**Comment by bdalomgir** [comment]
@argiepiano Not workings means Its not getting any accordion class i.e. jquery-ui-accordion. So total accordion functionality broke.

**Comment by argiepiano** [comment]
Does it work if instead of a Paragraph field for description you use a text field?

**Comment by argiepiano** [comment]
Also, which field has the accordion? Description? Or the field that points to the "Mosaic accordion" paragraph type? 
 Also, what type of paragraphs is Description pointing to? 

In other words, the more information you can provide, the better. It's hard to help without knowing all the details

**Comment by laryn** [comment]
@bdalomgir Is this still an issue in the latest version, or should we close it?

**Comment by bdalomgir** [comment]
@laryn Thanks so much for your reply.
I have experimented with the new version of the Paragraph module on different machines/Environments and found the same problem. You can reproduce the problem by following the steps.

**Steps are:**

1. Create a paragraph type called **Accordion one** with two field 
    - Accordion Title
    - Accordion body
2. Create another paragraph type called **Accordion two** with another two field
    -  Title
    - Body
3. You can edit any content type, go to the manage fields(`/admin/structure/types/manage/page/fields`), and create two-paragraph reference fields.
    - Accordion one refers to the **Accordion one(paragraph type)**
    - Accordion two refers to the **Accordion two(paragraph type)**
4. Then go to the manage display(`/admin/structure/types/manage/accordion/display/default`) and make these two field **Format** as **Paragraphs jQuery UI Accordion** and configured these fields.
5. Create content(/node/add/page) and add some accordion in **Accordion one** and **Accordion two** fields.
6. Now you can see the **Accordion one** is working perfectly and **Accordion two** is not.

**Comment by bdalomgir** [comment]
@laryn I have been debugging this issue for a while now and figured out that the problem I mentioned in my last comment is not related to this issue rather it's a Paragraphs jQuery UI Accordion issue. I found they use an attribute ID for each paragraph accordion and the id is the entity ID (nid) where the accordion is placed. I checked changing the ID to different text and both the accordion were working. We can close this issue. I might create another issue under paragraphs jquery UI accordion. Thanks 

New issue:
https://github.com/backdrop-contrib/paragraphs_jquery_ui_accordion/issues/7
